Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
+**/classes
```

Comment:
Actually,I found that the class files could be anywhere if you change the default output folder settings for specific java project.So do you think we should ignore all these class files wherever they are located in.






At 2016-05-01 16:57:41, "Brendan Forster" <notifications@github.com> wrote:


In Global/Eclipse.gitignore:

> @@ -49,3 +49,15 @@ local.properties
>  
>  # Code Recommenders
>  .recommenders/
> +
> +# Java class files
> +**/*.class
> +**/classes


I'm not sure on these rules - could you tell me a bit about where they come from?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub